ivadomed/main.py

```diff
@@ -276,10 +280,18 @@ def run_segment_command(context, model_params):
                 os.makedirs(pred_path)
 
             for pred, target in zip(pred_list, target_list):
-                filename = subject.split('.')[0] + target + "_pred" + \
-                           ".nii.gz"
+                filename = subject.split('.')[0] + target + "_pred" + ".nii.gz"
                 nib.save(pred, os.path.join(pred_path, filename))
 
+            # Get subject file extension and write PNG if not NifTI
+            # For Microscopy PNG/TIF files
```

Comment:
Very minor nitpick: These commends might be a little self-explanatory (the code itself describes almost exactly this).

Reply:
You are right, I updated the comment to only 1 line in 64fd62a. I only kept the info about which microscopy formats are supported and which are not (in relation to the microscopy BEP), this will help when implementing OMETIFF at some point.